library/src/pivotal-ui/components/forms/forms.scss

```diff
@@ -285,9 +285,48 @@ This form looks and behaves mostly like a content editable field
     cursor: pointer;
     outline: none;
     text-indent: -99999px;
-    width: $toggle-bg-width;
-    height: $toggle-bg-height;
-    border-radius: $toggle-bg-height;
+
```

Comment:
If at all possible, do not include actual pixel values in files other than pui-variables.scss, this makes maintenance much easier.

I'm hoping for `$toggle-bg-width-sm;`, `$toggle-bg-width-md`, etc. There will be some pixel values left over, hopefully those can be calculated, see below...